demisto_sdk/commands/doc_reviewer/rn_checker.py

```diff
@@ -0,0 +1,108 @@
+    click.echo('\n - Added the **reporter** and **reporter email** '
+               'labels to incidents that are created by direct messages.')
+    click.echo('\n - Improved implementation of the default value for the *fetch_time* parameter.')
+    click.secho('\n\nBug fixes examples:', fg='bright_cyan')
+    click.echo('\n - Fixed an issue where mirrored investigations contained mismatched user names.')
+    click.echo('\n - Fixed an issue with ***fetch-incidents***, which caused incident duplication.')
+    click.echo('\n - Fixed an issue in which the ***qradar-delete-reference-set-value*** command failed to '
+               'delete reference sets with the "\\" character in their names.')
+    click.secho('\n\nDocker Updates:', fg='bright_cyan')
+    click.echo('\n - Updated the Docker image to: *demisto/python3:3.9.1.15759*.')
+    click.secho('\n\nGeneral Changes:', fg='bright_cyan')
+    click.secho('Note: Use these if the change has no visible impact on the user, '
+                'but please try to refrain from using these if possible!', fg='yellow')
+    click.echo('\n - Maintenance and stability enhancements.')
+    click.echo('\n - Documentation and metadata improvements.')
+    click.echo('\n\nFor additional information see: https://xsoar.pan.dev/docs/documentation/release-notes')
+
+
+class ReleaseNotesChecker:
+    RN_TEMPLATES = {
+        'You can now', 'Added support for', 'Improved implementation', 'Fixed an issue',
+        'Updated the Docker image to', 'Added the', 'Maintenance and stability enhancements.',
+        'Documentation and metadata improvements.'
+    }
+
+    def __init__(self, rn_file: str, template_examples: bool = False):
+        if template_examples:
+            print_template_examples()
+
+        else:
+            with open(rn_file, 'r') as f:
+                self.file_path = rn_file
+                self.file_content = f.readlines()
+                self.notes = {}  # type:Dict
+
+    def add_note(self, line, note):
+        """Add note about an release notes line"""
+        if line in self.notes:
+            self.notes[line].append(note)
+        else:
+            self.notes[line] = [note]
+
+    def check_templates(self, line):
+        """Check that the given line fits one of our templates"""
+        if line.lower().startswith('added') and (line.lower().endswith('command:') or line.endswith('commands:')):
+            return True
+
+        good_line = False
+        for temp in self.RN_TEMPLATES:
+            if line.lower().startswith(temp.lower()):
+                good_line = True
+                break
+        return good_line
+
+    def print_notes(self):
+        """Print the review about the RN"""
+        for line in self.notes:
+            click.secho(f'\n - Notes for the line: "{line}"', fg='bright_red')
+            for note in self.notes[line]:
+                click.secho(f'   - {note}', fg='bright_red')
+
+    def check_rn(self) -> bool:
```

Comment:
check out the parsing of the RN files made in content: `Utils/release_notes_generator.py`

Reply:
took a look but could not find anything useful for this method - did you have anything in mind?